app/javascript/.storybook/main.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   // https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#correct-globs-in-mainjs
   stories: ['../**/__stories__/*.stories.@(mdx|jsx)'],
   addons: [
+    '@storybook/addon-controls',
```

Comment:
This is basically equivalent (eventually a replacement) for Knobs - which has been deprecated: 
- https://storybook.js.org/addons/@storybook/addon-controls
- https://storybook.js.org/addons/@storybook/addon-knobs

Reply:
Yeah. We weren't able to use this initially when we upgraded Storybook earlier this year. Nice to see that it works with Preact now. Mind creating an issue for that @ludwiczakpawel?